packages/story-editor/src/components/helpCenter/menu/tips.js

```diff
@@ -90,8 +90,12 @@ Tip.propTypes = {
   unread: PropTypes.bool,
 };
 
-const TIPS_ITERABLE = Object.entries(TIPS);
 export function Tips({ onTipSelect = () => {}, readTips }) {
+  const {
+    state: { tips },
+  } = useHelpCenter();
+  const TIPS_ITERABLE = Object.entries(tips);
```

Comment:
do we want to memoize this at all just to prevent any re-renders?

I'd also be open to camel casing it since it's not a module scoped constant

Reply:
Thanks, yes it should have been camel case, updated. 

Curious to know how memoizing it will prevent re-renders?

Reply:
Oh yea I guess you're right. It won't. never mind